core/src/main/java/org/polypheny/db/StatisticsManager.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019-2022 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db;
+
+import java.beans.PropertyChangeListener;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+public abstract class StatisticsManager<T extends Comparable<T>> implements PropertyChangeListener {
+
+    public static StatisticsManager<?> INSTANCE = null;
+
+
+    public static StatisticsManager<?> setAndGetInstance( StatisticsManager transaction ) {
```

Comment:
Why is this parameter called transaction?

Reply:
Changed name to statisticsManager.